libfabric.vcxproj

```diff
@@ -798,6 +798,7 @@
     <ClCompile Include="prov\efa\src\efa_mr.c" />
```

Comment:
The commit message `To make the clear more clear` . Is it `To make the code more clear`?

Reply:
Thank you! fixed in new revision.